src/xdsl/xdsl_opt_main.py

```diff
@@ -37,7 +38,7 @@ class xDSLOptMain:
 
     available_targets: Dict[str, Callable[[ModuleOp, IOBase], None]] = {}
     """
-    A mapping from target names to functions that serialize a ModuleOp into a 
```

Comment:
Could we remove those changes from this PR? This makes the blame quite unreadable.